platform/lib/build/samplesBuilder.js

```diff
@@ -160,7 +171,7 @@ class SamplesBuilder {
           for (const file of files) {
             stream.push(file);
           }
-
+        })).then(() => {
```

Comment:
We shouldn't combine then and await. While you're fixing this, can you also change  `.catch(...)` into a `catch` block into a

Reply:
Done in #2464.